### PR TITLE
CORE-331 Support pipe-delimited source files for de-identification an…

### PR DESCRIPTION
**Pull request by GrivIN**
…d ingestion

### CODE REVIEW

File: deident.py [hunk 1]
```diff
@@ -87,11 +87,10 @@ def deindent_csv(
     """Simple program that greets NAME for a total of COUNT times."""
     click.echo("start reading csv file")
```
Comment (tmcdonnell87) [inline deident.py]: not necessarily a CSV file anymore

File: db_processing/core/utils/data_utils.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            'primary_proc_cd': np.str,
+            'admitting_phys': np.str,
+            'attending_phys': np.str,
+            'discharging_phys': np.str,
+            'admitting_phys_NPI': np.str,
+            'attending_phys_NPI': np.str,
+            'discharging_phys_NPI': np.str,
+            'med_rec_no': np.str
+        },
+        converters=CONVERTERS
```
Comment (tmcdonnell87) [inline db_processing/core/utils/data_utils.py]: getting warning that both dtype and converter were specified for drg_no and admitting_phys columns

File: db_processing/core/utils/data_utils.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    )
+    return data
+
+
+def read_data(source_object, source_object_name):
+    if ".txt" in source_object_name:
```
Comment (GrivIN) [inline db_processing/core/utils/data_utils.py]: it does for sure (.txt anywhere in filename)

File: db_processing/core/utils/data_utils.py (hunk 2 above)
Comment (GrivIN) [inline db_processing/core/utils/data_utils.py]: should be fixed now

File: deident.py (hunk 1 above)
Comment (GrivIN) [inline deident.py]: changed to `start reading source file` (and `done reading source file`), also main function name changed

**Pull request merged**